### ISSUE TITLE
Shortcode quotes decode issue

### ISSUE THREAD

**Issue by dhanendrannewscorp** [Automattic/liveblog#560]
We have recently updated the Liveblog version to 1.9 and after that we are facing some issues with the shortcode rendering.

we found that the liveblog send the data as HTML encoded to the server. So we need to decode them before processing it. 
https://github.com/Automattic/liveblog/blob/master/classes/class-wpcom-liveblog-rest-api.php#L654

For example, we are trying to add a video embed into the entry box and it sends the data like

```

<p>[video width=&quot;1280&quot; height=&quot;720&quot; mp4=&quot;https://wwwexample.com/wp-content/uploads/2018/10/slow_motion_rain.mp4&quot;][/video]</p>

```

So we need to add `html_entity_decode()` [here](https://github.com/Automattic/liveblog/blob/master/classes/class-wpcom-liveblog-rest-api.php#L654)

**Comment by philipjohn** [comment]
Possibly related to #418 but needs investigating

**Comment by philipjohn** [comment]
Fixed by #570
